components/collector/pyproject.toml

```diff
@@ -50,3 +47,40 @@ module = [
     "defusedxml"
 ]
 ignore_missing_imports = true
+
+[tool.ruff]
+target-version = "py311"
+line-length = 120
+select = ["ALL"]
+ignore = [
+    "ANN001",  # https://beta.ruff.rs/docs/rules/missing-type-function-argument/ - too many untyped arguments atm to turn this rule on
```

Comment:
Isn't this a nice-to-have?

Reply:
Yes, but turning it on generates too many errors to fix right now.